samples/nft_deployment/README.md

```diff
@@ -0,0 +1,291 @@
+
+TODO define the right order
+
+We will cover the following topics:
+- Setup a Fabric network
+- Generate crypto material
+- Building the FSC node
+- Configuration
+- Run the node
+- Connecting to an existing Fabric network
+  - Lazy (use FSC Integration test network)
+- Building the Token Chaincode
+- Install & deploy Token Chaincode
+- Run the FSC nodes
+
+## Setup a Fabric network
+
+To illustrate how to use the Fabric Token SDK with an existing network, we will use microFab (TBD?!?!?)) as our Fabric network with the following configuration:
+1. Two organization: Org1 and Org2;
+2. Single channel;
+3. Org1 runs/endorses the Token Chaincode.
+
+We also need to add Idemix support to the network to allow business parties to hide their identities when submitting Fabric transactions.
+
+To start the network run:
+```bash
+just microfab
+```
+
+### Add Idemix Org
+
+We need to add an Idemix organization to allow business parties to sign Fabric transactions in an anonymous and unlikable way.
+
+#### Generate crypto material
+
+First step is to generate the Idemix Credential Issuer's key pair:
+
+```bash
+go install github.com/IBM/idemix/tools/idemixgen
+
+export IDEMIX_CRYTPO=$(pwd)/testdata/fabric/crypto
+
+idemixgen ca-keygen \
+  --output $IDEMIX_CRYTPO/peerOrganizations/idemixorg.example.com
+
+```
+
+Then, we need to generate the Idemix Credential Signer's key pairs for Alice and Bob, our business parties:
+
+```bash
+# alice
+idemixgen signerconfig \
+  --ca-input $IDEMIX_CRYTPO/peerOrganizations/idemixorg.example.com \
+  --output $IDEMIX_CRYTPO/peerOrganizations/org2.example.com/peers/alice.org2.example.com/extraids/idemix \
+  --admin \
+  -u idemixorg.example.com \
+  -e alice \
+  -r 140
+
+# bob
+idemixgen signerconfig \
+  --ca-input $IDEMIX_CRYTPO/peerOrganizations/idemixorg.example.com \
+  --output $IDEMIX_CRYTPO/peerOrganizations/org2.example.com/peers/bob.org2.example.com/extraids/idemix \
+  --admin \
+  -u idemixorg.example.com \
+  -e bob \
+  -r 150
+```
+
+TODO 
+
+### Add Idemix Org to channel
+
+1. Add idemixOrgs to network https://hyperledger-fabric.readthedocs.io/en/latest/idemix.html
+2. Update Channel configuration
+
+
+## Token Validation Chaincode
+
+We begin by building all components used in the nft sample.
+The Token SDK uses a Token Validation Chaincode with Fabric to translate Token Transaction into Fabric Transaction with RWSets.
+See more details in [TODO])().
+
+### Generate crypto material
+
+TODO: We need to generate crypto material for the Issuer, the Auditor, and the Token Owners.
+Issuer and Auditor are X509 certificates. For Token Owners, we need to setup an Idemix CA with curve `BN254`. 
+
+### Prepare public parameters
+
+We are using `tokengen` to create the public parameters used by the Token Validation Chaincode in our nft sample.
+
+tokengen gen fabtoken
+
+```bash
+go install github.com/hyperledger-labs/fabric-token-sdk/cmd/tokengen
+tokengen gen dlog \ 
```

Comment:
we need to add more parameters here: https://github.com/hyperledger-labs/fabric-token-sdk/blob/main/cmd/tokengen/README.md#tokengen-gen-dlog